pinc/bootstrap.inc

```diff
@@ -49,7 +49,7 @@ if (!defined('SKIP_DB_CONNECT')) {
     } catch (Exception $e) {
         // If we're in maintenance mode, don't die here - we'll more gracefully
         // error out later
-        if (!$maintenance) {
+        if (!SiteConfig::$maintenance) {
```

Comment:
The prefix is fine. As you pointed it, it is worth being explicit and the overhead seems small to me.